### ISSUE TITLE
"Share your data" workflow

### ISSUE THREAD

**Issue by josh-chamberlain** [Police-Data-Accessibility-Project/data-sources-app#40]
### Context

* This is a view in our app which guides people through the process of sharing their data with the public.
* Its audience is someone who is not a computer power-user, but has possession of public records they'd like to share.

### Requirements

1. guide people through publishing data somewhere (start with simplest first)
   - Use this doc as a starting point, and probably delete it: https://docs.pdap.io/activities/share-data/publish-data
   - if they already have a public URL, we can accept that
   - let people know they can publish this anywhere, but they will need an account to submit data they have published (a way of vouching for the veracity)
2. make sure to capture the link and keep them in our process
3. have them paste their public URL
   - Probably remove this doc, and redirect old links to the new workflow: https://docs.pdap.io/activities/share-data/share-published-data.
4. show the rest of the [data source submission form](https://docs.pdap.io/activities/share-data/contribute-data-sources) and have them submit that.
   - must be logged in and confirm that they are sharing public record

### Tests

* Anything we should test for? How should it be tested?

### Docs

* What docs should be updated? Link to related docs changes in the PR.

### Open questions

* Any uncertainties? Anything we're flexible about?

**Comment by maxachis** [comment]
@josh-chamberlain Revisiting this one -- do we still see this on the docket?

**Comment by josh-chamberlain** [comment]
@maxachis yes, though firmly in `future` and with the apt `needs refinement` label. I like the workflow of "just paste your URL and magic stuff happens" as well as helping people upload data.
